spatialmedia/gui.py

```diff
@@ -36,8 +36,10 @@
 path = os.path.dirname(sys.modules[__name__].__file__)
 path = os.path.join(path, '..')
 sys.path.insert(0, path)
-from spatialmedia import metadata_utils 
+from spatialmedia import metadata_utils
 
+SPATIAL_AUDIO_LABEL = "My video has spatial audio (ambiX ACN/SN3D format)"
+HEAD_LOCKED_STEREO_LABEL = "with head-locked stereo"
```

Comment:
Should we be consistent with "head locked" vs. "non-diegetic" terminology? What does YT documentation say?

Reply:
In favor of consistency. Opted for head-locked stereo (since it's a more user friendly term) and removed references to non-diegetic through-out.